app/editor/src/features/admin/cbra/CBRAReport.tsx

```diff
@@ -40,15 +40,19 @@ export const CBRAReport: React.FC<ICBRAReportProps> = ({ className }) => {
           onChange={(date) => setValues({ ...values, start: date })}
           disabled={!isCustomRange}
           selected={values.start}
+          width={FieldSize.Small}
```

Comment:
We have a bug with stretch.  The inputs will overlap each other.  Fixed sizes work.